src/EditLabelDialog/EditLabelDialog.jsx

```diff
@@ -197,11 +200,12 @@ export default function EditLabelDialog({
           onClick={handleSave}
           color="primary"
           disabled={
-            name.length === 0 ||
+            name.trim().length === 0 ||
             color.length === 0 ||
             !isLabelNameValid ||
             !hasChanged ||
-            !isLabelLengthValid
+            !isLabelLengthValid ||
+            !isDescriptionValid
```

Comment:
Me and @Sowbhagya-ipg have some reservations about this logic. Currently I can have a name of "       name" and you pass this on in the `onSave` and `onEdit` callbacks. We should be trimming before returning these callbacks so that while the component shows "     name", the callback receives "name".

The save button should not be disabled when a user has entered a description of "             " as it is not a required field, and it will be trimmed down to an empty description in the callbacks which is valid.